.env.dev

```diff
@@ -24,4 +24,6 @@ XCHAIN_ADDRESS=xchain.start
 CONTRACT_ADDRESS=pools.start
 CONTRACT_TABLE=pools
 
-POOL_BLACKLIST="[]"
\ No newline at end of file
+POOL_BLACKLIST="[]"
+
+BACKEND_ENDPOINT=http://localhost
```

Comment:
Is it possible to do comments here? Maybe just add a comment saying that this should be updated to point to the API server if possible.

Reply:
Fixed